v2/pubsub-cdc-to-bigquery/src/main/java/com/google/cloud/teleport/v2/templates/PubSubCdcToBigQuery.java

```diff
@@ -264,6 +264,12 @@ public static void main(String[] args) {
     Options options = PipelineOptionsFactory.fromArgs(args).withValidation().as(Options.class);
     BigQueryIOUtils.validateBQStorageApiOptionsStreaming(options);
 
+    if (options.getDeadLetterQueueDirectory() != null
```

Comment:
in case there were user specified empty string for deadLetterQueueDirectory, consider use `com.google.common.base.Strings.isNullOrEmpty` and treat empty string as null ?